src/core/image_resizer.js

```diff
@@ -88,6 +90,21 @@ class ImageResizer {
     );
   }
 
+  static get canUseImageDecoder() {
+    // TODO: remove the isChrome, once Chrome doesn't crash anymore with
+    // issue6741.pdf.
+    // https://issues.chromium.org/issues/374807001.
+    return shadow(
+      this,
+      "canUseImageDecoder",
+      // eslint-disable-next-line no-undef
+      this.#isChrome || typeof ImageDecoder === "undefined"
+        ? Promise.resolve(false)
+        : // eslint-disable-next-line no-undef
+          ImageDecoder.isTypeSupported("image/bmp")
+    );
+  }
```

Comment:
Nit: Can we please move this up, such that this getter isn't placed "between" the closely related `MAX_DIM` and `MAX_AREA` getters?